types/webdriverio/index.d.ts

```diff
@@ -34,6 +35,9 @@ declare namespace WebdriverIO {
     }
 
     interface ParsedCssProperty {
+        hex: string;
+        alpha: number;
+        rgba: string;
```

Comment:
as per https://github.com/webdriverio/webdriverio/blob/master/lib/helpers/parseCSS.js#L28 another possible value is `rgb` (without alpha)